style.css

```diff
@@ -0,0 +1,100 @@
+@import 'https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap';
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  background: #f1f1f1;
+}
+
+.main-header {
+  height: 50px;
+}
+
+.menu {
+  position: absolute;
+  left: 80%;
+  top: 4%;
+  font-size: 24px;
+}
```

Comment:
- [ ] In order to layout the menu please use the property `flex` on the menu parent container as stated in the project requirements 👍🏼, instead of setting a position of absolute since this will mess up the page layout and it's not useful in this case.

![Screenshot from 2023-03-21 23-24-20](https://user-images.githubusercontent.com/68030297/226755630-757d30da-88f4-4c85-9108-79e9d89dd194.png)

Visit the following link for more information about [When do I use absolute positioning?](https://teamtreehouse.com/community/when-do-i-use-absolute-positioning-and-when-do-i-use-relative-positioning)